bbc/simorgh
# Update HIndi and Persian podcast promo and Nepali nav

Resolves JIRA: n/a

Overall changes
======
Update the in article podcast promo for Hindi an Persian to promote their new podcast. Update Nepali navigation to remove link to elections

Code changes
======

1. Update podcastPromo section in /src/app/lib/config/services/hindi.ts
2 Update podcastPromo section in /src/app/lib/config/services/persian.ts
3. Remove second link in the navigatio section of  /src/app/lib/config/services/nepali.ts

Testing
======
1. Open /hindi/india-60426858 check that podcast promo links to /hindi/podcasts/p0fmrg25 brand image, title and description should match the screenshot below
2. Open persian/arts-52166891 check that podcast promo links to /persian/podcasts/p05gyy09 brand image, title and description should match the screenshot below
3. Open the Nepali front page, there should be no link wiht link text नेपाल निर्वाचन २०७९ poiting to /nepali/topics/cp2d78r6qppt

![hindi_podcast](https://github.com/bbc/simorgh/assets/10046386/68277d8c-390c-4a1b-a87a-5e69003473f1)
![persian_podcast](https://github.com/bbc/simorgh/assets/10046386/cd79c795-f60d-4e17-8c40-446a49822f0e)
![nepali_nav](https://github.com/bbc/simorgh/assets/10046386/1b83b326-4ebc-411f-b678-90902e1c4b29)




Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
